lib/location/scan_wifi.c

```diff
@@ -160,7 +160,7 @@ int scan_wifi_init(void)
 
 	wifi_iface = NULL;
 #if defined(CONFIG_WIFI_NRF700X)
-	wifi_dev = device_get_binding("wlan0");
+	wifi_dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_wifi));
```

Comment:
This should be `ncs_location_wifi` to cater for all Wi-Fi chips.

Reply:
As mentioned offline, the idea of having `ncs_location_wifi` sounds nice. But I'm not an expert on device tree issues and how they are composed so I'll leave it to you if you want to make this or not. It could be done later too as we are in a hurry now.

Reply:
Sure, thanks.